data/src/main/java/ca/bc/gov/data/datasource/local/DependentsLocalDataSource.kt

```diff
@@ -16,4 +16,7 @@ class DependentsLocalDataSource @Inject constructor(
 
     suspend fun insertDependents(list: List<DependentEntity>) =
         dependentsDao.insert(list)
+
+    suspend fun findDependent(phn: String) =
```

Comment:
I highly recommend using complete names for variables. Just for readabilities purposes.

Reply:
I agree we should avoid acronyms to improve readability, but in this context: PHN (patient health number) it's common sense for both mobile and backend